packages/frontend/src/pages/UserActivity.styles.ts

```diff
@@ -44,12 +47,12 @@ export const BigNumber = styled.div`
     font-size: 4em;
     line-height: 1.196em;
     font-weight: 500;
-    color: ${Colors.DARK_GRAY4};
+    color: #383e47;
```

Comment:
From blueprint colors. I figured it was ok to hardcode these for now.

Reply:
can we use mantine equivalent of it?